keps/sig-release/3031-signing-release-artifacts/README.md

```diff
@@ -102,6 +100,17 @@ discussions about how to utilize the existing Google infrastructure as well as
 consider utilizing keyless signing via workload identities. Nevertheless, this
 KEP focuses more on the "What" aspects rather than the "How".
 
+### Alpha implementation
+
+The alpha phase of the proposal is about signing the official Kubernetes
+container images and providing a minimum infrastructure to achieve that goal.
+
+### Beta graduation
+
+Graduation the KEP to beta means that we will now sign all artifacts which got
+created during the release process. This includes binary artifacts, source code
+tarballs, documentation, provenance data and the SBOM.
```

Comment:
I would leave the provenance attestations out of the beta criteria.

My reasoning is that during beta, we will finish testing and signing the artifacts. In this case, we need to test the missing piece: signing files (binaries, tarballs)

Signing the attestations requires a bit more work which we can do while we test signing plain files. We can sign the SBOMs now as plain files but we should later consider distributing them as attestations in the images too.

Reply:
Done, added a paragraph to explicitly exclude the provenance data.